src/Kernel/client/kernel.ts

```diff
@@ -31,20 +31,43 @@ class Kernel {
                 console.log("my message received", message);
 
                 //create buttons as DOM objects in order to attach unique event handlers
-                let state_div = message.content.state.div_id;
+                let state: DisplayableState = message.content.state;
+                let state_div = state.div_id;
                 if (state_div != null) {
                     let div = document.getElementById(state_div);
                     if (div != null) {
+                        let button1 = document.createElement("button");
```

Comment:
I recognize this button is only used for a short period before getting added to `div`, but it may still be helpful to give it a more descriptive variable name.

Reply:
changed the names of all the buttons to something more descriptive!